### DISCUSSION TITLE
How to use different logo icons for dark and light modes

### DISCUSSION THREAD

**Discussion by sakurayun** [squidfunk/mkdocs-material#5324] [Q&A] [answered]
How to use different logo icons for dark and light modes
I tried https://github.com/squidfunk/mkdocs-material/discussions/2729 But it cannot take effect

**Comment by squidfunk** [comment, marked as answer]
it's hard to give you any advise, without knowing what you did exactly. You need to follow the steps in #2729, override the theme and add additional CSS. All code that is necessary was posted in #2729.
